functionary/ui/views/schedules/update.py

```diff
@@ -0,0 +1,85 @@
+from django.http import HttpRequest, HttpResponse, HttpResponseRedirect, QueryDict
+from django.shortcuts import get_object_or_404, render
+from django.urls import reverse
+
+from core.auth import Permission
+from core.models import Environment, Function, ScheduledTask
+from ui.forms import ScheduledTaskForm, TaskParameterForm
+from ui.views.schedules.utils import get_crontab_schedule
+from ui.views.view_base import PermissionedFormUpdateView
+
+
+class ScheduledTaskUpdateView(PermissionedFormUpdateView):
+    model = ScheduledTask
+    form_class = ScheduledTaskForm
+    template_name = "forms/schedules/scheduling_create_or_update.html"
+
+    def get_success_url(self) -> str:
+        return reverse("ui:schedule-list")
```

Comment:
Now that you have the detail view, should that be where the success_url points?

Reply:
Fixed